src/main/java/net/atos/zac/flowable/CMMNService.java

```diff
@@ -109,7 +110,8 @@ public void startCase(final Zaak zaak, final Zaaktype zaaktype, final Zaakafhand
                     .variable(VAR_ZAAK_UUID, zaak.getUuid())
                     .variable(VAR_ZAAK_IDENTIFICATIE, zaak.getIdentificatie())
                     .variable(VAR_ZAAKTYPE_UUUID, zaaktype.getUUID())
-                    .variable(VAR_ZAAKTYPE_OMSCHRIJVING, zaaktype.getOmschrijving());
+                    .variable(VAR_ZAAKTYPE_OMSCHRIJVING, zaaktype.getOmschrijving())
+                    .variable(VAR_ZAAKTYPE_DOMEIN, zaakafhandelParameters.getDomein());
```

Comment:
oei ... dit is wellicht niet een goed idee.
Want als iemand het domein van een zaaktype veranderd dan zullen alle reeds gestarte zaken bij het oude domein blijven.
Dat lijkt mij niet de bedoeling ......